### PR TITLE
docs: Document all Espresso extensions

### PR DESCRIPTION
**Pull request by mykola-mokhnach**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -332,14 +332,498 @@ execTimeout | number | no | Timeout used to wait for a server reply to the given
...
+index | int | no | If the given locator matches multiple elements then only the element located by this index will be selected for the interaction, otherwise the method will be applied to all found elements.  Indexing starts from zero | 1
+
+#### Returns
+
+The result of the selected action applied to found elements. If index is provided then the array will only contain one item. If the index is greater than the count of found elements then an exception will be thrown.
+
+### mobile: uiautomatorPageSource
+
+Allows to retrieve accessibility elements hierarchy tree with UiAutomator](https://developer.android.com/training/testing/ui-automator) framework. The extension calls [dumpWindowHierarchy](https://developer.android.com/reference/androidx/test/uiautomator/UiDevice#dumpwindowhierarchy_1) under the hood.
```
Comment (KazuCocoa) [inline README.md:745]: `[` before _UiAutomator_? (a broken markdown syntax)

File: README.md [hunk 2]
```diff
@@ -332,14 +332,498 @@ execTimeout | number | no | Timeout used to wait for a server reply to the given
...
+
+Allows to execute a limited set of [UiAutomator](https://developer.android.com/training/testing/ui-automator) commands to allow out-of-app interactions with accessibility elements.
+
+#### Arguments
+
+Name | Type | Required | Description | Example
+--- | --- | --- | --- | ---
+strategy | string | yes | UiAutomator element location strategy. The following strategies are supported: "clazz", "res", "text", "textContains", "textEndsWith", "textStartsWith", "desc", "descContains", "descEndsWith", "descStartsWith", "pkg" | desc
+locator | string | yes | Valid UiObject2 locator value for the given strategy | 'my description'
+action | string | yes | The action name to perform on the found element. The following actions are supported: "click", "longClick", "getText", "getContentDescription", "getClassName",  "getResourceName", "getVisibleBounds", "getVisibleCenter", "getApplicationPackage", "getChildCount", "clear", "isCheckable", "isChecked", "isClickable", "isEnabled", "isFocusable", "isFocused", "isLongClickable", "isScrollable", "isSelected" | isEnabled
```
Comment (KazuCocoa) [inline README.md:736]: 👍

**Review by KazuCocoa — APPROVED** [review]
Awesome 🎉

File: README.md (hunk 1 above)
Comment (mykola-mokhnach) [reply README.md]: fixed

**Pull request merged**